lib/configs/api-gateway/schema.ts

```diff
@@ -488,48 +488,33 @@ export const articleListSchema: JsonSchema = {
                             "author": {
                                 type: JsonSchemaType.STRING,
                             },
-                            "summary": {
-                                type: JsonSchemaType.STRING,
-                            },
                             "created_at": {
                                 type: JsonSchemaType.STRING,
                             },
-                            "updated_at": {
+                            "update_at": {
                                 type: JsonSchemaType.STRING,
                             },
-                            "tags": {
-                                type: JsonSchemaType.ARRAY,
-                                items: {
-                                    type: JsonSchemaType.STRING,
-                                },
-                            },
                             "src": {
                                 type: JsonSchemaType.STRING,
                             },
                         },
                         required: [
-                            "category",
+                            "type",
                             "title",
                             "author",
-                            "summary",
                             "created_at",
-                            "updated_at",
-                            "tags",
+                            "update_at",
                             "src",
                         ],
                     },
                 },
                 "size": {
                     type: JsonSchemaType.INTEGER,
                 },
-                "offset": {
```

Comment:
Offset is needed, otherwise the frontend wouldn't know how to get the next batch.

Reply:
The front end wants to use an integer offset, but not a pointer for simplicity. Another thing is that, if the frontend doesn't want to use pages like linked list, they may want to jump between pages but not visit one by one. We can make it possible to give the frontend a page index to do pages at first (maybe one more step),  but we also need to scan over the table to make indexes. So, since also adds to the frontends' work, Hao suggests I not implement the real page for now.